.github/workflows/run-integration-tests.yml

```diff
@@ -76,7 +76,10 @@ jobs:
         strategy:
             matrix:
                 php: [8.1, 8.0, 7.4, 7.3]
-                laravel: [8.*]
+                laravel: [8.*, 9.*]
+                exclude:
+                  - php: [7.4, 7.3]
```

Comment:
The matrix still generates jobs for PHP 7.4 and Laravel 9, see https://github.com/barryvdh/laravel-ide-helper/actions/runs/1692091626

Maybe the `[…, …]` is not supported and you've to manually list each?
```
- php 7.3
  laravel: 9.*
- php 7.4
  laravel: 9.*
```
Just a guess though

Reply:
ok i've updated matrix to suggested syntax.. sorry it's my first time with github actions syntax

Reply:
np!

yeah it can be a PITA, also since you're first time contributor, we've to manually enabled running the workflows everytime 😅 (I just did, let's wait for the result)

Reply:
We're one step further, next problem: orchestra/benchmark also needs to have adapted version constraints, see https://github.com/barryvdh/laravel-ide-helper/runs/4803289512?check_suite_focus=true#step:5:20

I see they have not yet made a new 7.* release at https://github.com/orchestral/testbench-core but their branch is also updated. Maybe we can use `dev-master` there or we need to wait.

FTR: If you feel overwhelmed / frustrated, don't be: I can take it from here later if you want.

Reply:
@mfn if it's not a problem please take it from here because i don't wan't to break something... thx for the support!  :)

Reply:
Let's wait for the 9.x release to run the ingration tests.